### PR TITLE
Feature/dr 1561 require ready domains

### PR DESCRIPTION
**Pull request by cristiangiagante**
## Background
  
Styles issue detected testing on QA
  
## Changes done
-Add require domain validation in plans view
-Styles moved to _settings.scss
-Fix styles to be responsive with mobile resolutions
  
## Notes
Tested on Chrome, Edge, Firefox (last versions) in desktop and mobile mode.  

## Demo
 ISSUE:
image

FIX:
image

### CODE REVIEW

File: src/wwwroot/styles/views/_settings.scss [hunk 1]
```diff
@@ -620,6 +620,67 @@
...
+      bottom: 100%;
+      left: 24%;
+      border: solid transparent;
+      content: " ";
+      position: absolute;
+      border-bottom-color: $suit-accent-color;
+      border-width: 9px;
+      margin-left: -9px;
+    }
+    ol{
```
Comment (mvillaseco) [inline src/wwwroot/styles/views/_settings.scss]: ol is for ordered lists, im not sure if this is the case, maybe UL is more common to use in this cases, just consider it

File: src/wwwroot/styles/views/_settings.scss [hunk 2]
```diff
@@ -620,6 +620,67 @@
...
+      padding-left: 15px;
+      margin-bottom: 0px;
+      li{
+        border-left: 0px;
```
Comment (mvillaseco) [inline src/wwwroot/styles/views/_settings.scss]: those borders :(, if it has to be like this, then leave it, we can fix that later.

File: src/wwwroot/styles/views/_settings.scss (hunk 1 above)
Comment (cristiangiagante) [reply src/wwwroot/styles/views/_settings.scss]: Is an orderer list based on the mocks.

File: src/wwwroot/styles/views/_settings.scss (hunk 2 above)
Comment (cristiangiagante) [reply src/wwwroot/styles/views/_settings.scss]: I attached an image to explain why I need to override this value explicitly.
image

**Pull request merged**